week5/will/8/index.js

```diff
@@ -0,0 +1,21 @@
+const will = {
+    name: "Will Wu",
+    birthplace: "China",
+    age: "25",
+    isGraduated: "false",
+    education: "The future sphere",
+    favoriteFood: ["Cheese burger", "Beef over rice", "Taco"],
+    favoriteStar: {
+        name: "Kobe Bryant",
+        profession: "Basketball Star",
+        age: "41",
+    },
+};
+
+window.alert(will.name);
+
+document.write(will.age);
+
+document.getElementById("food").innerHTML = will.favoriteFood;
+
+console.log(will.favoriteStar);
```

Comment:
you are console logging an object, you have to select which property to console log

Reply:
fixed